newrelic.md

```diff
@@ -0,0 +1,25 @@
+---
+order: 50
+---
+# New Relic for DSpace
+
+The task involves downloading the New Relic agent and attaching it to Tomcat. Consult the [New Relic Java Agent](https://docs.newrelic.com/docs/agents/java-agent/installation/install-java-agent) insturctions if you run into any issues. 
+
+## Steps
+
+- Download and unzip the New Relic Java agent to `/etc/newrelic`.
+- Make `tomcat` user/group the owner (as on dspace-staging) and verify file permissions.
+- Open `newrelic.yml`: update the `license_key` (from account settings in New Relic), `app_name` (e.g., dspace_prod), and `log_file_path` (e.g., `/var/log/newrelic`) references.
```

Comment:
Can we specify `dspace-prod` rather than `dspace_prod`? The other tiers we've named have used dashes, rather than underscores.